sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -50,50 +53,53 @@ dependencies {
     exclude group: 'io.grpc', module: 'grpc-grpclb'
   }
   compile library.java.google_cloud_core
-  compile(library.java.google_cloud_core_grpc) {
```

Comment:
I'm very hesitant about removing so many `compile` dependencies after we discovered BEAM-11732. I think we should create a JIRA task to complete later, add them all back, mark them as permitUnusedDeclared, and leave a comment linking to the JIRA.